easybuild/easyconfigs/h/heFFTe/heFFTe-2.3.0-cpeAMD-22.12-ROCm.eb

```
@@ -0,0 +1,42 @@
+easyblock = 'CMakeMake'
+
+name = 'HeFFTe'
+version = '2.3.0'
+versionsuffix = '-ROCm'
```

```suggestion
versionsuffix = '-rocm-5.2'
```
As we have multiple version of ROCm installed I think it's better to include ROCm version so that we can have the same version of a package builds for multiple ROCm versions

---

I was not sure what type of naming would be practical here... it is rocm version independent actually, maybe `-GPU` would be better?

---

I see your point now, sorry.